api/src/Microsoft.Azure.IIoT.Api/src/Publisher/Models/EngineConfigurationApiModel.cs

```diff
@@ -43,10 +43,10 @@ public class EngineConfigurationApiModel {
         public int? MaxMessageSize { get; set; }
 
         /// <summary>
-        /// Max IoT D2C outgress message buffer size
+        /// Max IoT D2C egress message queue
```

Comment:
>essage queue [](start = 32, length = 12)

is this a count?

Reply:
It's a count: MaxEgressMessageQueue * 256 KB would be the max. size.